src/iop/spots.c

```diff
@@ -685,6 +686,58 @@ void gui_cleanup(dt_iop_module_t *self)
   free(self->gui_data);
   self->gui_data = NULL;
 }
+
+void init_key_accels (dt_iop_module_so_t *module)
+{
+  dt_accel_register_iop (module, FALSE, NC_("accel", "spot circle tool"),   0, 0);
```

Comment:
Why second param to dt_accel_register_iop(), local is FALSE here?
So that it is not local to the module?